### ISSUE TITLE
Train a set of models

### ISSUE THREAD

**Issue by JJanowiak** [IBM/pychemex#6]
set-up a ML pipeline to train a set of ML models based on the sets of features calculated.
Details to be confirmed later.

**Comment by Alex-AMC** [comment]
[[Descriptors]] have been calculated, all 208 of them. 
[[Fragments]] are included amonst these as seen in [RDkit](http://rdkit.org/docs/source/rdkit.Chem.Fragments.html)

[[Models]] to develop - an array, free for all.

#### Starting 
- [[Multivariate Linear Model]] 
- [[Random Forest]]
- [[gxboostedTrees]]
- [[Logistical Regression]]

---

#### [[Neural Network]] 

- scikit-learn - have a play. 
- 

### [[Pre-Processing required]]
- Training : Test Split 85:15
- Scaling would be required for most of them (minus the RF)
	
### [[Optimisation]]
- [[Feature Reduction]]
- [[Cross Validation]]


### Carry out work in [[Jupyter]]
- If model works, save as a [[pickle]]



#### Notes:
- Scikit-learn allows you to write pipelines which allows you to do [[Pre-Processing]] -> [[Model Building ]] -> [[Validation]] 
- What would be useful to know can we think of any new hypotheses (?)
-

**Comment by Alex-AMC** [comment]
Trained multivariant regression model and random forest regressor with feature selection optimisation and hypermeter optimisation done, the best result was a 93% accuracy with an error of 29K error.
